docs/specs/src/hwp_1_intro.md

```diff
@@ -0,0 +1,212 @@
+Holochain
+
+Distributed Coordination by Scaled Consent not Global Consensus
+
+Abstract
+========
+
+In this paper we present a frame and a specification for a generalized  
+system for large scale distributed coordination that does not rely on
+global consensus. We start with some axioms; we proceed to describe the
+consequences of these axioms, looking at the problem through the lens of
+scaling consent; we present an informal description of the system; we
+provide a more formal outline of the necessary integrity guarantees and
+system components needed to deliver those guarantees; and finally we
+conclude with a high-level yet sufficiently detailed technical
+specification of what's needed to implement such a system.
+
+Introduction
+============
+
+Preamble
+--------
+
+The first version of the [[Holochain white
+paper]{.underline}](https://github.com/holochain/holochain-proto/blob/whitepaper/holochain.pdf)
+took on a very formal approach to describing generalized distributed
+computation schemes and carefully compared the approach taken by
+Blockchain based systems with the approach taken by Holochain as well as
+providing formal reasoning for the benefits of Holochain's agent centric
+approach.
+
+In this second version, we describe Holochain on its own terms by
+providing the highest level view of our approach and our starting
+axioms. At the next level of detail we provide an informal system
+description using the metaphor of games. Then we dive into deeper detail
+with a formal system description including the specific context and
+assumptions we come from, the integrity guarantees Holochain offers, and
+a formal state model and discussion of security and safety concerns.
+Finally at the most detailed level we provide an implementation
+specification in Appendix A.
+
+Axioms
+------
+
+We begin by stating our axioms[^1] regarding the nature of coordination:
+
+1.  **Coordination arises from agents** starting from the same
+  ground-rules and acting as soon as each party in any interaction
+  can confirm that each other\'s actions confirm that actions or
+  interactions conform to those ground rules. (Thus, in our frame,
```

Comment:
Having reflected about this for a while I'm thinking that in "real, fuzzy life" there are gradients in how subjectively held rules map onto each other. And coordination starts (or fails) to arise, as soon as at least one agent subjectively concludes based on past behavior of the other agents involved that the gradients between his subjectively held rules and those of the other agents involved are small enough that its worth laying an initial "trace of actions" for others to pick up on.

That Holochain, on the layer of the software code, embodies a means for *each* party in *any* interaction to be able to confirm that each other's actions confirm that actions or interactions conform to those ground rules is then rather a specific attempt for a solution to this more fuzzy problem of how to get agents start playing together according to shared rules. But I think I don't consider it to be appropriate to take as an axiom.
For example, it may start to show its limitations when "non-techie" people interact with Holochain that have no internal maps whatsoever that they could use to subjectively verify for themselves with high level of confidence how and whether those rules are *actually* embodied in the software.

Reply:
> to confirm that each other's actions confirm that actions or interactions conform to those ground rules

Hmm, there's that clause again, so the fact that you're phrasing it this way makes me think I'm missing something and not able to parse it. Diving into the English grammar of this clause: what does it mean for "actions to confirm that actions conform"? The similarity of "confirm" and "conform" notwithstanding, this seems confusingly self-referential, and maybe that's on purpose, but it's breaking my brain.

Reply:
I see that there's a typo there.  I think it should read:

```
**Coordination arises from agents** starting from the same
  ground-rules and any single agent's ability to act as soon as they have 
  confirmed to their satisfaction that other agents existing actions
  conform to those ground rules.
```

Reply:
clairfied